source/lex.tex

```diff
@@ -1793,48 +1800,13 @@
 \indextext{\idxcode{0}|seealso{zero, null}}%
 \indextext{\idxcode{0}!string terminator}%
 \indextext{\idxcode{0}!null character|see {character, null}}%
-After any necessary concatenation, in translation phase
-7\iref{lex.phases}, \tcode{'\textbackslash 0'} is appended to every
-\grammarterm{string-literal} so that programs that scan a string can find its end.
-
-\pnum
-Escape sequences and \grammarterm{universal-character-name}{s} in non-raw string literals
-have the same meaning as in \grammarterm{character-literal}s\iref{lex.ccon}, except that
-the single quote \tcode{'} is representable either by itself or by the escape sequence
-\tcode{\textbackslash'}, and the double quote \tcode{"} shall be preceded by a
-\tcode{\textbackslash},
-and except that a \grammarterm{universal-character-name} in a
-UTF-16 string literal may yield a surrogate pair.
-\indextext{string!\idxcode{sizeof}}%
-In a narrow string literal, a \grammarterm{universal-character-name} may map to more
-than one \tcode{char} or \tcode{char8_t} element due to \defnadj{multibyte}{encoding}. The
-size of a \tcode{char32_t} or wide string literal is the total number of
-escape sequences, \grammarterm{universal-character-name}{s}, and other characters, plus
-one for the terminating \tcode{U'\textbackslash 0'} or
-\tcode{L'\textbackslash 0'}. The size of a UTF-16 string
-literal is the total number of escape sequences,
-\grammarterm{universal-character-name}{s}, and other characters, plus one for each
-character requiring a surrogate pair, plus one for the terminating
-\tcode{u'\textbackslash 0'}.
-\begin{note}
-The size of a \tcode{char16_t}
-string literal is the number of code units, not the number of
-characters.
-\end{note}
-\begin{note}
-Any \grammarterm{universal-character-name}{s} are required to
-correspond to a code point in the range
-$[0,$ $\mathrm{D800})$ or $[\mathrm{E000},$ $\mathrm{10FFFF}]$ (hexadecimal)\iref{lex.charset}.
-\end{note}
-The size of a narrow string literal is
-the total number of escape sequences and other characters, plus at least
-one for the multibyte encoding of each \grammarterm{universal-character-name}, plus
-one for the terminating \tcode{'\textbackslash 0'}.
+In translation phase 7\iref{lex.phases},
+after adjacent string-literals are concatenated,
```

Comment:
string-literal should be  \grammarterm'ed

Reply:
fixed